observation_portal/settings.py

```diff
@@ -442,7 +447,7 @@ def get_dramqtiq_broker_url():
 TEST_RUNNER = 'observation_portal.test_runner.MyDiscoverRunner'
 
 try:
-    from local_settings import *  # noqa
+    from .local_settings import *  # noqa
```

Comment:
Does this change anything - I would think both reference a local settings file in your root directory...

Reply:
The original does not work for me but nevertheless, I had not intended to commit that. Reverted.